sections/git_branch.zsh

```diff
@@ -25,6 +37,26 @@ spaceship_git_branch() {
   git_current_branch="${git_current_branch#heads/}"
   git_current_branch="${git_current_branch/.../}"
 
+  # Build commit SHA info if enabled
+  if [[ $SPACESHIP_GIT_BRANCH_SHOW_COMMIT == true ]]; then
+    local git_commit_preffix="$SPACESHIP_GIT_BRANCH_COMMIT_PREFIX$SPACESHIP_GIT_BRANCH_COMMIT_SYMBOL"
+    local git_commit_sha=$(git rev-parse --short HEAD 2> /dev/null)
+    local git_commit_info="$git_commit_preffix$git_commit_sha$SPACESHIP_GIT_BRANCH_COMMIT_SUFFIX"
+  fi
+
+  git_current_branch+="$git_commit_info"
+
+  # Build tag info if enabled
+  if [[ $SPACESHIP_GIT_BRANCH_SHOW_TAG == true ]]; then
+    local git_tag_preffix="$SPACESHIP_GIT_BRANCH_TAG_PREFIX$SPACESHIP_GIT_BRANCH_TAG_SYMBOL"
+    local git_tag_suffix="$SPACESHIP_GIT_BRANCH_TAG_SUFFIX"
+    local git_tag_name=$(git describe --tags --exact-match $git_commit_sha 2>/dev/null)
+    local git_tag_info="$git_tag_preffix$git_tag_name$git_tag_suffix"
```

Comment:
Why not use `_TAG_SUFFIX` directly here as previously noted by @olets ?

Reply:
You're right. I didn't noticied!